tensorboard/webapp/feature_flag/views/feature_flag_page_component.ng.html

```diff
@@ -30,6 +30,12 @@ <h2 class="warning">WARNING: EXPERIMENTAL FEATURES AHEAD!</h2>
       Only flags with non default values are sent to the backend.
     </div>
   </ng-container>
+  <ng-container *ngIf="showFlagsFilter">
+    <div class="message">
+      Feature Flags are filtered to only show features containing
+      "{{showFlagsFilter}}"
+    </div>
+  </ng-container>
```

Comment:
This is an extremely basic message and isn't super obvious. I don't think it matters too much but it might make sense to wrap this in an `h2` or something similar.